tests/utils.py

```diff
@@ -42,6 +42,10 @@
 POSTGRES_PASSWORD = os.environ.get("SYNAPSE_POSTGRES_PASSWORD", None)
 POSTGRES_BASE_DB = "_synapse_unit_tests_base_%s" % (os.getpid(),)
 
+# When debugging a specific test, it's occasionally useful to write the
+# DB to /tmp and query it with the sqlite CLI.
+SQLITE_TEST_DB_LOCATION = os.environ.get("SYNAPSE_TEST_SQLITE_DB_LOCATION", ":memory:")
```

Comment:
I debated if this should default to `None` so we could encapsulate the magic `":memory:"` string in a single spot, what do you think?